src/ConfigurationTrait.php

```diff
@@ -83,13 +83,14 @@ public function getConfig($forceRefresh = false)
         $connectionConfig = ConnectionManager::config($connection);
         $adapterName = $this->getAdapterName($connectionConfig['driver']);
 
+        $templatePath = Plugin::path('Migrations') . 'src' . DS . 'Template';
```

Comment:
Can we have the DS attached here at the end? Paths in CakePHP usually always end this way.
Further down then

    'file' => $templatePath . 'Phinx' . DS . 'create.php.template'